packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -72,7 +71,7 @@ const PersonalDetailsForm = props => {
     }, [no_confirmation_needed, setStatus, status]);
 
     const getNameAndDobLabels = () => {
-        const is_asterisk_needed = is_svg || is_mf || is_rendered_for_onfido || is_qualified_for_idv;
+        const is_asterisk_needed = is_svg || is_mf || is_rendered_for_onfido || is_rendered_for_idv;
```

Comment:
a lot of conditions & seems like the * is required in every scenario

Reply:
Seems like we can just remove this function and directly provide the label to the respective fields

Reply:
Good point.. I cant think of any scenario that is not covered